core/extensions/impl/crypto_extension.cpp

```diff
@@ -196,24 +196,25 @@ namespace kagome::extensions {
 
   namespace {
     auto encode_result = [](auto &&value) -> common::Buffer {
+      static auto logger =
+          common::createLogger("CryptoExtension::scale_encode");
```

Comment:
I think is better to lazy creation in **if-then** block, especially it's single using instance.

Reply:
static is better, since it saves time